### ISSUE TITLE
[General] Issue management

### ISSUE THREAD

**Issue by iosdeveloper** [pebble/pebblekit#33]
Please go over all the open issues and close those fixed as soon as you're releasing a new version of the SDK. Right now there are some issues that are no longer relevant.
You could even do so by including
`Fixes #XX`
in the commit message.
Thanks!

**Comment by sarfata** [comment]
You are right. We have several issue management which may explain why some are still opened here. I will look into all the issues and update the status. Thanks for the feedback.
